app/src/main/java/com/thailam/piggywallet/ui/wallet/WalletActivity.java

```diff
@@ -78,7 +78,10 @@ private void initToolbar() {
     private void initFab() {
         FloatingActionButton fab = findViewById(R.id.fab_add);
         fab.setOnClickListener(v -> {
-            Intent intent = new Intent(this, AddWalletActivity.class);
+//            Intent intent = new Intent(this, AddWalletActivity.class);
+//            startActivity(intent);
+              //TODO: change later
+            Intent intent = new Intent(this, WalletDetailActivity.class);
```

Comment:
Phải define static method getWalletDetailIntent(Context context) trong class WalletDetailActivity và gọi nó để lấy intent nhé, xem thêm tại convention 2.11: https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#211-argument-in-activity-and-fragment

Reply:
em test vào màn hình khác tí quên không xóa